packages/pn-commons/src/components/CustomDatePicker.tsx

```diff
@@ -19,15 +16,17 @@ const locales: { [key: string]: Locale } = {
 
 export type DatePickerTypes = Date | null;
 const CustomDatePicker = (
-  props: DesktopDatePickerProps<DatePickerTypes> &
+  props: DatePickerProps<DatePickerTypes> &
     React.RefAttributes<HTMLDivElement> & { language?: string }
 ) => {
   const language = props.language ? props.language : 'it';
   return (
     <LocalizationProvider dateAdapter={AdapterDateFns} adapterLocale={locales[`${language}`]}>
-      <DesktopDatePicker
+      <DatePicker
         {...props}
+        desktopModeMediaQuery="@media (pointer: fine)"
```

Comment:
remove this property because it is the default value

Reply:
done